helper/utils.js

```diff
@@ -0,0 +1,10 @@
+export const PreProcessSocialName= (socialobject) =>{
+   
+    if(socialobject.url.includes('linkedin') && socialobject.displayName.includes('company/')){
+        const newName = socialobject.displayName.split('/')[1]
+        return `in/${newName}`
```

Comment:
Use only `newName`. Please don't add `in/`

Reply:
Ok